src/cloud/client/cli-pre-signed-url.ts

```
@@ -0,0 +1,119 @@
+      expires: option({
+        long: "expires",
+        type: string,
+        defaultValue: () => "3600",
+        defaultValueIsSerializable: true,
+      }),
+      now: option({
+        long: "now",
+        type: {
+          async from(str): Promise<string> {
+            const decoded = new Date(str);
+            if (isNaN(decoded.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)